nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/StandardNiFiWebConfigurationContext.java

```diff
@@ -377,7 +380,7 @@ public ComponentDetails setAnnotationData(final NiFiWebConfigurationRequestConte
             });
 
             final ProcessorDTO processor;
-            if (properties.isClustered()) {
+            if (StandardNiFiWebConfigurationContext.this.properties.isClustered()) {
```

Comment:
@olegz thanks but I think IntelliJ might have to take the credit for this one :)